### ISSUE TITLE
Event Streams across Period Boundaries

### ISSUE THREAD

**Issue by haudiobe** [Dash-Industry-Forum/AdInsertion#45]
We need more study on Events across Event Boundaries. 
Two issues:
-	Event Stream continuity across Boundaries
-	Events with duration across Period boundary
One possible solution: Global and Local Event Streams. 
Add a note here and continue to address it.

**Comment by ZmGorynych** [comment]
Global streams would be helpful.
SCTE 35 cue messages can be "open" for much longer than the segments are available (e.g. program boundaries), and may span several periods

**Comment by RufaelDev** [comment]
I am unsure how the global events would work as you will lose synchronization to the media timeline in case periods are added, modified etc. During call Thomas raised similar suggestion introducing event stream continuity. Signalling continuity for events makes sense, might be the default even. What kind of events would be signalled in global event streams ?

**Comment by ZmGorynych** [comment]
Global events need to be timed off @availabilityStartTime, /methinks.
The use case we have is "open" SCTE 35 events from previous periods where
all segments are already out of availability window. Conceptually this can
be viewed as "event history" of events with presentation time earlier than
the current period.
Additional use case can be when event duration spans multiple periods, b/c
a period was created due to temporary loss of input transcoder input for a
few 5 seconds.

**Comment by technogeek00** [comment]
The MPEG ad-hoc is picking up the global event stream work, should we defer this until that work is completed and pull in the result?

**Comment by ZmGorynych** [comment]
Sure!

**Comment by RufaelDev** [comment]
How about just recommending use of presentationTimeOffset in event streams, and re-signalling events in each period ? adding presentationTimeOffset to EventStream was an important fix in amendment 5. Creating a new global event stream causes more confusion around this topic. I am also ok with having event stream continuity cross boundaries

**Comment by RufaelDev** [comment]
To be constructive and try to progress this some proposed text for the document: 

In case opportunity data crosses a next period, the following is recommended:

An EventStream Element with same schemeIdUri, value and timescale attributes is present in the next Period. 
The EventStream Element in the next period shares a common anchor, such as Unix Epoch 1-1-1970 or another shared anchor. This anchor is used to set the EventStream@presentationTimeOffset to align the EventStream to the start time of the period.
Thus a period that start 10000 ticks on the timescale after the anchor would contain an EventStream@presentationTimeOffset of 10000. Events from previous periods that are still active in the current period are then signalled again
using the same presentationTime, id and value as in one of the previous periods. 
In other words, they are repeated in this Period. In case the opportunity data is no longer active, 
such re-signalling is not needed.

**Comment by haudiobe** [comment]
DASH-IF IOP 2021/05/14:
- For MPD Event: 
   - Repeat the Event, signal the `@presentationTimeOffset` - similar as a media segment.
   - need to signal that the Event is continued. Is this not clear by the Event Stream?
   - We may need to define an Event Stream identifier that allows to indicate that the Event@ids continue across.
